.gitignore

```diff
@@ -5,3 +5,12 @@
 /ecs-init-*
 *.swp
 /ecs.conf
+/.rpm-done
+/.srpm-done
+/BUILD
+/RPMS
+/SOURCES
+/SRPMS
+/ecs-init.spec
+/sources.tgz
+ecs-init/version/version.go
```

Comment:
One approach I've seen used for golang based CLI releases use a previous commit (tagged or otherwise) so that the dirty repository is only dirtied by the change to the `version.go`. However, the approach @samuelkarp mentions is my preference using something along the lines of https://github.com/cogolabs/lgrep/blob/c4efafa3ad84faa51df0bc48bc5c4e702c9b4b8f/Makefile#L4 or https://github.com/coreos/etcd/blob/31e3899663522556e06933297e12fd030efae488/build#L15 where the Version variables are overriden at build time. I would also rather not eliminate this file entirely.